front/app/components/HookForm/InputMultilocWithLocaleSwitcher/index.tsx

```diff
@@ -34,10 +34,14 @@ const InputMultilocWithLocaleSwitcher = ({ name, ...rest }: Props) => {
   );
 
   // Select the first error messages from the field's multiloc validation error
-  const errorMessage = Object.values(
+  const validationError = Object.values(
     (errors[name] as Record<Locale, FieldError> | undefined) || {}
   )[0]?.message;
 
+  const apiError =
+    (errors[name]?.error as string | undefined) &&
+    ([errors[name]] as unknown as CLError[]);
```

Comment:
Not sure if possible - but is there a way to avoid the "as" typecasting in lines 38 - 43?

Reply:
In this particular case it would be difficult - the format of our own errors and the type that RHF expects are somewhat incompatible so to make the back-end validation work without introducing breaking changes to the existing forms, we need to cast it for the moment. Definitely something we can improve in the future, though.